client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import FilePicker from 'components/file-picker';
+import DropZone from 'components/drop-zone';
+import { localize } from 'i18n-calypso';
+import notices from 'notices';
```

Comment:
Is it possible that the old `notices` are deprecated and that we should use either the [`Notices`](https://wpcalypso.wordpress.com/devdocs/design/notices) or [`GlobalNotices`](https://wpcalypso.wordpress.com/devdocs/design/global-notices) component instead?